### PR TITLE
Add an extra end-to-end example with stackcollapse

### PR DESCRIPTION
**Pull request by mhansen**
And add some words around 'stackcollapse' to help people find this repo if they're looking for scripts to deal with the output of the `stackcollapse-*` scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,23 +15,29 @@ go install github.com/felixge/pprofutils/cmd/...@latest
...
 text2pprof < example.txt > example.pprof
 ```
 
 Warning: Converting from pprof to text is lossy. Only the first sample type will be converted, file names, lines, labels, and more will be dropped. Patches to make things less lossy would be welcome, but please open an issue first to discuss.
 
-Create a delta profile that contains the difference `heap-b.pprof - heap-a.pprof`:
+Convert a Linux `perf.data` profile to `pprof`, via Brendan Gregg's [`stackcollapse-perf.pl`](https://github.com/brendangregg/FlameGraph/blob/master/stackcollapse-perf.pl) script:
 
+```bash
+stackcollapse-perf.pl < perf.data | text2pprof > perf.pprof
```
Comment (felixge) [inline README.md:35]: NIT: This makes it seem like `stackcollapse-perf.pl` can process a binary perf data file, which is not the case. `stackcollapse-perf.pl` can process the output of `perf script`. So maybe this snippet should read:

```suggestion
perf script | stackcollapse-perf.pl | text2pprof > perf.pprof
```

**Review by felixge — APPROVED** [review]
LGTM. Let me know if you think the NIT should be addressed, but I'm also happy to merge this as-is. Thanks a lot!

**Comment by mhansen** [comment]
D’oh. You’re right of course! Ill fix it up

**Comment by mhansen** [comment]
Fixed, thanks for the suggestion. That's the first one I've applied through the github UI; was really easy.

**Comment by felixge** [comment]
@mhansen thanks! Yeah, the suggestion feature is one of the few things I like about GitHub PRs 😅

**Pull request merged**